### ISSUE TITLE
Increase package_info_plus dependency to >= 6.0.0

### ISSUE THREAD

**Issue by marcelpinto** [wiredashio/wiredash-sdk#340]
**Describe the bug**
I would like to update to the package_info_plus v6.0.0 but wiredash is pinned to a lower version. Is there any specific constraint to limit it to <v6?

**Stacktrace**
Because wiredash 2.1.0 depends on package_info_plus >=3.0.0 <6.0.0 and no versions of wiredash match >2.1.0 <3.0.0, wiredash ^2.1.0 requires package_info_plus >=3.0.0 <6.0.0.

**Comment by passsy** [comment]
Thanks for the heads-up. I wasn't aware of the new major version.
I haven't tested it yet, but don't expect any breaking API changes. It should be compatible.

For now, you can use a `dependency_override`.

```yaml
dependency_override
  # Workaround for https://github.com/wiredashio/wiredash-sdk/issues/340
  package_info_plus: ^6.0.0
```

I'll bump it for the next release

**Comment by sebastianbuechler** [comment]
> Thanks for the heads-up. I wasn't aware of the new major version. I haven't tested it yet, but don't expect any breaking API changes. It should be compatible.
> 
> For now, you can use a `dependency_override`.
> 
> ```yaml
> dependency_override
>   # Workaround for https://github.com/wiredashio/wiredash-sdk/issues/340
>   package_info_plus: ^6.0.0
> ```
> 
> I'll bump it for the next release

There's even another major release of the package: https://pub.dev/packages/package_info_plus/changelog @passsy Can you please upgrade again?

**Comment by ciriousjoker** [comment]
The relevance of this is somewhat pushed by `firebase_core` requiring the higher `package_info_plus` version since >=2.29.0

**Comment by passsy** [comment]
Fix has been released as [`2.1.1`](https://pub.dev/packages/wiredash/versions/2.1.1)
